### ISSUE TITLE
Require php-http/async-client-implementation dependency repository

### ISSUE THREAD

**Issue by bojanpejic** [getsentry/sentry-php#781]
I was trying to get latest Sentry version `2.0` with Composer, before doing it I've actually removed old Sentry version `1.10`.

When I've run `composer require sentry/sentry:2.0` I've got the following output:
```
  Problem 1
    - Installation request for sentry/sentry 2.0 -> satisfiable by sentry/sentry[2.0.0].
    - sentry/sentry 2.0.0 requires php-http/async-client-implementation ^1.0 -> no matching package found.

Potential causes:
 - A typo in the package name
 - The package is not available in a stable-enough version according to your minimum-stability setting
   see <https://getcomposer.org/doc/04-schema.md#minimum-stability> for more details.
 - It's a private package and you forgot to add a custom repository to find it

Read <https://getcomposer.org/doc/articles/troubleshooting.md> for further common problems.
```

Ok, I've checked and the requested dependency `php-http/async-client-implementation` doesn't seam to exist. I'm not sure what happened with it, but it isn't listed between other repositories on [php-http](https://github.com/php-http).

Any suggestions what might went wrong?

**Comment by stayallive** [comment]
Hey @bojanpejic, thanks for jumping in with 2.0!

I will not explain here too since I'll probably do a bad job, please check out the [install instructions](https://github.com/getsentry/sentry-php#install) in the readme that explains what that package is and how you can install it if you want.

tl;dr: use `composer require sentry/sdk:2.0` instead of requiring `sentry/sentry` 👍

**Comment by CharlotteDunois** [comment]
The tl:dr would be that it's not an actual package but a virtual one (package A: "I need something that implements package XY", package B: "I provide an implementation for package XY"). For example the HTTPlug cURL client provides it. <https://github.com/php-http/curl-client/blob/master/composer.json#L45-L48>

**Comment by bojanpejic** [comment]
Right, I've totally missed the change of package name and updated documentatio, my bad. @CharlotteDunois thanks for the explanation for virtual package!
